app/controllers/videos_controller.rb

```diff
@@ -1,5 +1,8 @@
 class VideosController < ApplicationController
+  before_action :require_user, except: [:index]
```

Comment:
How come you are using `except` here, then manually doing pretty much the same thing in the `index` action?